e2etests/bats-tests.sh

```diff
@@ -785,6 +785,30 @@ get_value_from() {
   [[ "${count}" == "2" ]]
 }
 
+@test "restart-policy" {
+  tmp="tests/checks/restart-policy.yaml"
+  cmd="${KUBE_LINTER_BIN} lint --include restart-policy --do-not-auto-add-defaults --format json ${tmp}"
+  run ${cmd}
+
+  message1=$(get_value_from "${lines[0]}" '.Reports[0].Object.K8sObject.GroupVersionKind.Kind + ": " + .Reports[0].Diagnostic.Message')
```

Comment:
IMHO, it would be good to add the object name to the messages in order to ease debugging.
As far as I can see, there are more objects in the test file than there are messages/reports here. While it is easy to deduce which ones should be present in the report, I would prefer to have the debugging information state more explicitly what objects are expected to be reported by the linter.

Reply:
Adjusted

Reply:
I think the valid object in the test file did belong there too.
What I was thinking for the test was something along the lines (if the report objects expose the required fields):
```
Deployment fire-deployment-never: object has a restart policy defined with 'Never' but the only accepted restart policies are '[Always OnFailure]'
```